submissions/Mifaresss/js_dom/script.js

```diff
@@ -0,0 +1,126 @@
+"use strict"
+
+const cards = [
+	{
+		id: 1,
+		header: 'Мастера меча онлайн',
+		img: './images/sao.jpg',
+		subHeader: 'Краткое описание',
+		description: `Начало двадцать первого века, японский гений создал виртуальный мир «Айнкрад», релиз которого ждало огромное количество людей по всему миру. Геймеры получили возможность полного погружения. Новинка стала самой ожидаемой и десять тысяч образцов с молниеносной скоростью исчезли с прилавков. Все вошли в виртуальную реальность, растворившись в ней и собрались в Городе. Появился создатель. Огромная черная тень пояснила участникам, что вернуться в реальный мир возможно только пройдя всю игру. Но есть один жестокий момент, виртуальная смерть означает и смерть в реальной жизни и пройти её наобум не получится. После громких слов создатель испарился, оставив игроков одних.
+		<p class="title-description__text">Реакция на сообщение оказалась разной. Началось отчаяние, кто-то объединился с другими, а некоторые, как герой Кадзуто, пошли по пути одни. Киригая сирота и поэтому попав в этот мир, он ни чего не теряет. Он бета-тестер и зная всё об этой игре, быстро завоевал популярность у игроков. Герой понимает, что без взаимодействия с другими ему не выбраться и приходится налаживать контакты с остальными игроками. А пока идёт бой на мечах, всем предстоит увидеть ещё немало удивительных вещей.</p>`

```

Comment:
Avoid Html tags. Add paragraphs to the array.  
```
[{
   ....
   description: [ 'one', 'two', 'three', ..... ],
   ....
}]
```

Reply:
I do not understand how to change the function so that it works correctly (

Reply:
done